### PR TITLE
Chapters 1-9/Day 0-2 Data structures code

### PR DESCRIPTION
**Pull request by ssingal05**

### CODE REVIEW

**Comment by davidabrahams** [comment]
Needs a `README.md` telling them exactly which sections they need to fill in, and from what chapters. Check the readme on my pull request for an example

File: day02/src/Profiler.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import org.jfree.chart.axis.NumberAxis;
+import org.jfree.chart.plot.PlotOrientation;
+import org.jfree.chart.plot.XYPlot;
+import org.jfree.data.xy.XYDataItem;
+import org.jfree.data.xy.XYSeries;
+import org.jfree.data.xy.XYSeriesCollection;
+import org.jfree.ui.ApplicationFrame;
+import org.jfree.ui.RefineryUtilities;
+
+/**
```
Comment (thecardkid) [inline day02/src/Profiler.java]: Remove (question mark)

File: day02/src/MyLinkedListTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/**
```
Comment (thecardkid) [inline day02/src/MyLinkedListTest.java]: Empty comments

File: day02/src/MyLinkedList.java [hunk 3]
```diff
@@ -0,0 +1,277 @@
+/**
```
Comment (thecardkid) [inline day02/src/MyLinkedList.java]: Empty

File: day02/src/MyArrayListTest.java [hunk 4]
```diff
@@ -0,0 +1,301 @@
+/**
```
Comment (thecardkid) [inline day02/src/MyArrayListTest.java]: Empty

File: day02/src/MyArrayList.java [hunk 5]
```diff
@@ -0,0 +1,245 @@
+/**
```
Comment (thecardkid) [inline day02/src/MyArrayList.java]: Empty

File: day01/src/MyArrayList.java [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+	public T get(int index) {
+		if (index < 0 || index >= size) {
+			throw new IndexOutOfBoundsException();
+		}
+		return array[index];
+	}
+
+	@Override
+	public int indexOf(Object target) {
+		//TODO
```
Comment (thecardkid) [inline day01/src/MyArrayList.java]: Can you change this to say `// TODO: Implement this method` or sth equivalent?

File: day01/src/MyArrayList.java [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+	}
+
+	@Override
+	public boolean retainAll(Collection<?> collection) {
+		throw new UnsupportedOperationException();
+	}
+
+	@Override
+	public T set(int index, T element) {
+		//TODO
```
Comment (thecardkid) [inline day01/src/MyArrayList.java]: See above

File: day02/src/MyArrayList.java (hunk 5 above)
Comment (thecardkid) [inline day02/src/MyArrayList.java]: Are all of `MyArrayList.java`, `MyArrayListTest.java`, and `SelectionSort.java` required to complete the LinkedList homework? I'm skeptical. If not, can you remove unnecessary Java classes from the day02/src/ directory? We should keep the codebase for each lab to an absolute minimum.

File: code/lab01/build.xml [hunk 8]
```diff
@@ -73,7 +73,7 @@
         <junit fork="yes" printsummary="withOutAndErr">
```
Comment (thecardkid) [inline code/lab01/build.xml]: Can you remove this change from the commit/PR?

**Pull request closed without merging**